cmd2/parsing.py

```diff
@@ -61,7 +61,7 @@ def __new__(cls,
                 *,
                 raw: str = None,
                 command: str = None,
-                args: str = None,
+                arg_list: List[str] = None,
```

Comment:
To me it seems like it would be more for the ``Statement`` class to initialize the ``str`` attributes to an empty string instead of ``None``.  For the ``List`` attributes, it seems like it better to check if they are ``None`` in the ``__new__`` body and if so set them to an empty list.

@kotfu What would you think about making this change?

Reply:
Yup, it's better as empty strings and lists, instead of `None`.